### ISSUE TITLE
w-confirm forces new line

### ISSUE THREAD

**Issue by sbeaver28** [antoniandre/wave-ui#106]
```html
<w-confirm icon="mdi mdi-delete">
  Delete
</w-confirm>

<w-confirm icon="mdi mdi-pencil">
  Edit
</w-confirm>`

Stacks the two icons on top of each other. I would expect them to appear side-by side.

**Comment by sbeaver28** [comment]
Wrapping both w-confirms in a <w-flex wrap> puts both icons back on the same line. Maybe that is the intended behavior?

**Comment by antoniandre** [comment]
Hi @sbeaver28, thanks for reporting.
You're right that's not the desired behavior and it should now be fixed in version 3.1.3.
https://antoniandre.github.io/wave-ui/release-notes

Please reopen if not solved :)
